checkita-core/src/main/scala/ru/raiffeisen/checkita/storage/Tables.scala

```diff
@@ -138,6 +150,17 @@ class Tables(val profile: JdbcProfile) {
       executionDate
     ) <> (ResultCheckLoad.tupled, ResultCheckLoad.unapply)
   }
+
+  class ResultJobConfigTable(tag: Tag, schema: Option[String])
```

Comment:
Rename to JobStateTable and inherit directly from DQTable. Change table name to `job_state` as per comments for db migration scripts.